test/11_array.jl

```diff
@@ -132,6 +132,38 @@ for (A, V, M) in ((NominalArray, NominalVector, NominalMatrix),
         @test x[3] === x.pool.valindex[1]
         @test levels(x) == ["e", "a", "b", "c"]
 
+        x = V{String, R}(a)
```

Comment:
Why not use the same `x` as above? That's the approach I've taken, which is more likely to discover subtle corruption bugs than simple operations on newly-created arrays.